src/content/pages/credits.mdx

```diff
@@ -25,4 +25,7 @@ Thanks for the gorgeous photographs, friends. 📸❤️
 - Microphone photo by [Tim Arterbury](https://unsplash.com/@tim_arterbury?utm_medium=referral&amp;utm_campaign=photographer-credit&amp;utm_content=creditBadge) on Unsplash.
 - Typewriter photo by [David Klein](https://unsplash.com/@diklein?utm_medium=referral&amp;utm_campaign=photographer-credit&amp;utm_content=creditBadge) on Unsplash.
 
+## Legalese
+
+Apple, the Apple logo, Mac, macOS, Mac App Store, and Swift are trademarks of Apple Inc.
```

Comment:
Should we quote these?
```suggestion
"Apple", the Apple logo, "Mac", "macOS", "Mac App Store", and "Swift" are trademarks of Apple Inc.
```

Reply:
[Apple's guidelines](https://www.apple.com/legal/intellectual-property/guidelinesfor3rdparties.html) just say:

> ______ and______ are trademarks of Apple Inc.

...and where I've seen them used on third-party sites, I haven't noticed quotes.

From Apple's own site: https://support.apple.com/en-gb/guide/ipad/ipad9a87cada/ipados

> Apple, the Apple logo, AirDrop, AirPlay, AirPods, AirPrint, Animoji, Apple Music, Apple News, Apple Pay, Apple Pencil, Apple TV, Apple Wallet, Apple Watch, Face ID, FaceTime, Finder, Flyover, Guided Access, Handoff, iMessage, iPad, iPad Air, iPad mini, iPadOS, iPad Pro, iPhone, iPod, iPod touch, iTunes, Keychain, Keynote, Lightning, Mac, macOS, Magic Mouse, Magic Trackpad, Numbers, Pages, Photo Booth, Retina, Retina HD, Safari, Siri, Smart Cover, Spotlight, Touch ID, TrueDepth, True Tone, and watchOS are trademarks of Apple Inc., registered in the U.S. and other countries.